views/albums.hbs

```diff
@@ -0,0 +1,18 @@
+<body class="albumsBody">
+    <div class="container">
+        <ul class="albumsUl">
```

Comment:
Only `<li>` can be nested directly after a `<ul>` element.

You can nest a `<div>` inside the  nested`<li>`
```
<ul class="albumsUl">
  {{#each albums}}
  <li>
            <div class="albumsCard">
                <img src={{this.images.2.url}} alt="albumcover" class="albumsImg">
                <div class="albumsText">
                    <span class="albumsLi">{{this.name}}</span>
                    <br>
                    <button class="albumsButton"><a href="/tracks/{{this.id}}" class="page3a">View Tracks</a></button>
                    </div>
                 </div>
                </li>
            {{/each}}
        </ul>